src/Message/ApplePayAuthorizeRequest.php

```diff
@@ -411,10 +413,14 @@ public function sendData($data)
 
         /** @var EventDispatcherInterface|null $eventDispatcher */
         $eventDispatcher = $this->httpClient->getEventDispatcher();
+        $request_name = 'startSession';
```

Comment:
could you make the name of this one the object name too so it matches the other?